src/classlibnative/inc/nlsinfo.h

```diff
@@ -145,53 +86,7 @@ class COMNlsInfo {
 
     static void QCALLTYPE nativeNormalizationInitNormalization(int NormForm, BYTE* pTableData);
```

Comment:
The code using it is kind of clear about that:

https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/Text/Normalization.Windows.cs#L48

but I'll add a comment anyway